src/Core/src/VisualDiagnostics/Extensions/VisualDiagnosticsExtensions.Impl.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Maui
+{
+	public static class VisualDiagnosticsExtensions
```

Comment:
Is this the correct filename? The `.Impl`? Will it be included every platform? Should it be `.Standard` or something. I see it is not a partial class, so I am wondering why the compiler is not upset.

Reply:
Ahh, didn't know that existed! Updating.